src/WCSimDetectorConfigs.cc

```diff
@@ -1012,12 +1011,16 @@ void WCSimDetectorConstruction::SetHyperK_HybridmPMT_WithOD_Realistic_Geometry()
 
   // TEST WLS collection for stacking action
   G4String WLSType="";
-  if( HKFD_OD_WLS_PLATE_TYPE == "EljenEJ286" )
+  if( HKFD_OD_WLS_PLATE_TYPE == "EljenEJ286" ){
 	WLSType = "EljenEJ286";
-  else if( HKFD_OD_WLS_PLATE_TYPE == "Kuraray" )
+	WCODWLSPlatesThickness   = 1.*cm; //
+  }else if( HKFD_OD_WLS_PLATE_TYPE == "Kuraray" ){
 	WLSType = "Kuraray";
-  else if( HKFD_OD_WLS_PLATE_TYPE == "Inr" )
+	WCODWLSPlatesThickness   = 0.6*cm; //
+  }  else if( HKFD_OD_WLS_PLATE_TYPE == "Inr" ){
 	WLSType = "Inr";
+	WCODWLSPlatesThickness   = 0.7*cm; //
+  }
```

Comment:
`WCODWLSPlatesThickness` is used to calculate `WCODDiameter`, but isn't set until after it's used

Basically, need to move this block up to line 981 again